pkg/controller/configuration/ensure.go

```diff
@@ -701,12 +701,17 @@ func (c *Controller) ensureTerraformPlan(configuration *terraformv1alpha1.Config
 
 		// @step: lets build the options to render the job
 		options := jobs.Options{
-			AdditionalJobSecrets: state.additionalJobSecrets,
+			AdditionalAnnotations: state.provider.GetAnnotations(),
+			AdditionalJobSecrets:  state.additionalJobSecrets,
 			AdditionalLabels: utils.MergeStringMaps(
-				utils.MergeStringMaps(configuration.GetLabels(), map[string]string{
+				// thats a lot of labels!
+				c.JobLabels,
+				state.provider.GetLabels(),
```

Comment:
should this be JobLabels()?

Reply:
Updated both names ... the to the `jobs.Options` and the `c.JobLabels,` -> `c.ControllerJobLabels,` (i.e.  the ones defined globaly to all jobs regardless of provider ..

Reply:
still wondering if it should be state.provider.JobLabels() tho?